azure-simple/packer/ansible-is/docs/Pattern1.md

```diff
@@ -0,0 +1,36 @@
+# Pattern 1 - HA clustered deployment of WSO2 Identity Server
+
+## Deployment diagram
+![HA clustered deployment of WSO2 Identity Server](images/Deployment-pattern-1-diagram.png)
+
+## Changing hostname and ports
+1. Uncomment the parameters in the `is_2.yml` and give the values for both `is_1.yml` and `is_2.yml`, following the [documentation](https://docs.wso2.com/display/IS560/Setting+Up+Deployment+Pattern+1) to deploy pattern 1.
```

Comment:
Doc is pointing to the 560 version. Wouldn't it be better if we can rename the file `is_2.yml` to `is_pattern_2.yml ` assuming `2` represents the pattern 2 of IS.

Reply:
Doc version updated. As per my understanding  refers the hostname(hosts) as is_1 and is_2  in the Ansible playbook and not the pattern number.